src/js/expanderExpanded.js

```diff
@@ -54,7 +57,7 @@ function mdExpanderExpandedDirective($mdUtil, $animateCss) {
       })
       .start()
       .then(function () {
-        if (isHeightSet !== undefined) {
+        if (isHeightSet === true) {
```

Comment:
To me it communicates that the type might be unknown or there's a possibility of it being not a bool. Personal preferences. Not a show stopper.